docs/guide/docs/main/code.md

```diff
@@ -1,15 +1,17 @@
 ---
 id: code
-title: Actions & Hooks
+title: Built-In Functionalities
```

Comment:
Why the change of title?

Reply:
Because I added Skills to the section. Should I give it as "Actions, Hooks and Skills" ?

Reply:
Or maybe split the topic into three?

Reply:
''Coding Actions, Hooks & Skills'' What do you think?